es/03_3_Configurando_Su_Billetera.md

```diff
@@ -0,0 +1,136 @@
+
+Con una sola dirección en la mano, puede pasar directamente a la siguiente sección y comenzar a recibir fondos. Sin embargo, antes de llegar allí, discutiremos brevemente los otros tipos de direcciones que encontrará en el futuro y hablaremos sobre algunos otros comandos de billetera que quizás desee usar en el futuro.
+
+### Conociendo sus direcciones de Bitcoin
+
+Hay tres tipos de direcciones de Bitcoin que puede crear con el comando RPC `getnewaddress`. Utilizará una dirección `legacy` (P2PKH) aquí, mientras que se moverá a una dirección SegWit (P2SH-SegWit) o a una dirección Bech32 en [§4.6: Creating a Segwit Transaction](04_6_Creating_a_Segwit_Transaction.md)
+
+Como se señaló anteriormente, la base de una dirección de Bitcoin es una clave pública: alguien envía fondos a su clave pública y luego usa su clave privada para canjearla. 
+¿Fácil? Excepto que poner su clave pública allí no es del todo seguro. Por el momento, si alguien tiene su clave pública, entonces no puede recuperar su clave privada (y por lo tanto sus fondos); esa es la base de la criptografía, que utiliza una función de trampilla para garantizar que solo pueda pasar de la clave privada a la pública, y no al revés. 
+Pero el problema es que no sabemos lo que deparará el futuro. Excepto que sabemos que los sistemas de criptografía eventualmente se rompen por el avance incesante de la tecnología, por lo que es mejor no poner claves públicas en crudo en la red, para asegurar sus transacciones en el futuro.
+
+Las transacciones clásicas de Bitcoin crearon direcciones P2PKH que agregan un paso criptográfico adicional para proteger las claves públicas.
+
+> :book: ***Qué es una dirección heredada (P2PKH)?*** Esta es una dirección heredada del tipo utilizado por la naciente red de Bitcoin. Lo usaremos en ejemplos para las próximas secciones. Se llama dirección Pay to PubKey Hash (o P2PKH) porque la dirección es un hash de 160 bits de una clave pública. El uso de un hash de su clave pública como su dirección crea un proceso de dos pasos en el que gastar los fondos que necesita para revelar tanto la clave privada como la clave pública, y aumenta la seguridad futura en consecuencia. Este tipo de dirección sigue siendo importante para recibir fondos de personas con software de billetera desactualizado.
```

Comment:
- `***¿Qué es una dirección heredada (P2PKH)?***`
- `Esta es una direccion de tipo Legacy, del tipo de aquellas usadas en los origenes de la red Bitcoin.`
- `El uso de un hash de su clave pública como su dirección crea un proceso de dos pasos en el que para gastar los fondos necesita  revelar tanto la clave privada como ...`

Reply:
- The first bullet point refers to the missing opening question mark.
- I think that Legacy is a better expression of the meaning in the second bullet point.
- In the third bullet point the key is the use of `para` inside `en el que para gastar los fondos`. I think that it improves the translation.